src/signature.js

```diff
@@ -51,6 +51,14 @@ export class Signature {
   armor(config = defaultConfig) {
     return armor(enums.armor.signature, this.write(), undefined, undefined, undefined, config);
   }
+
+  /**
+   * Returns an array of KeyIDs of all of the issuers who created this signature
+   * @returns {Array<KeyID>} The Key IDs of the signing keys
+   */
+  getSigningKeyIDs() {
+    return this.packets.filter(packet => packet.tag !== enums.packet.marker).map(packet => packet.issuerKeyID)
```

Comment:
And then finally, I would prefer to select the signature packets explicitly, rather than the other way around. See `Message#getSigningKeyIDs` for inspiration. (Also, there's a final semicolon missing.)

Reply:
I don't quite understand how `Message#getSigningKeyIDs` relates to my solution.

Reply:
figured it out